ghcide/src/Development/IDE/GHC/Compat/Units.hs

```diff
@@ -324,7 +325,7 @@ moduleUnit =
     Module.moduleUnitId
 #endif
 
-filterInplaceUnits :: [UnitId] -> [PackageFlag] -> ([UnitId], [PackageFlag])
+filterInplaceUnits :: NE.NonEmpty UnitId -> [PackageFlag] -> ([UnitId], [PackageFlag])
```

Comment:
I would use types unqualified, hope it is the pattern for other data types

Reply:
Ditto, not sure this one should change.